pkg/integrations/install/install.go

```diff
@@ -27,6 +27,6 @@ import (
 	// v2 integrations
 	//
 
-	_ "github.com/grafana/agent/pkg/integrations/v2/agent" // register agent
-	_ "github.com/grafana/agent/pkg/integrations/v2/eventhandler"
+	_ "github.com/grafana/agent/pkg/integrations/v2/agent"             // register agent
+	_ "github.com/grafana/agent/pkg/integrations/v2/app_o11y_receiver" // register app_o11y_receiver
```

Comment:
You accidentally removed the other integration here :)

Reply:
- [ ] oops, bad merge. Restore